repo_policy_compliance/check.py

```diff
@@ -157,6 +157,11 @@ def target_branch_protection(
     protection = branch.get_protection()
 
     pull_request_reviews = protection.required_pull_request_reviews
+    if pull_request_reviews is None:
+        return Report(
+            result=Result.FAIL,
+            reason=(f"{FAILURE_MESSAGE}pull request reviews are not required, {branch_name=!r}"),
```

Comment:
How about `pull request reviews must be enabled` or something suggestive of user's action rather than describing the state of the repository?

Reply:
All the error messages are describing the problem, it would be good to add something like a unique code for each problem and then have documentation on how to resolve those problems. Including instructions with how to solve the problem with every error would be quite complicated and you can't include things like screenshots etc

Reply:
I'll add an issue for that

Reply:
https://github.com/canonical/repo-policy-compliance/issues/583